oaknational/Oak-Web-Application
# LESQ-969: Implement OakPagination component on OWA with SEO

## Description

Music year: 1993 - https://www.youtube.com/watch?v=zkHOVJINRD8

- Create and implement `PaginationHead` Component to add pagination metadata
- Imports OakPagination from CompLib and adds to Unit, Lessons, Search and Blogs/Webinar pages
- Amends usePagination hook, to return props for OakPagination component
- Updates test files props in line with OakPagination  comp
- Removes old Pagination component from OWA

## Issue(s)

Fixes #LESQ-969

## How to test

1. Go to https://deploy-preview-2664--oak-web-application.netlify.thenational.academy

2. Navigate to any page with pagination i.e. Search, Blogs/Webinars, certain lesson and unit pages
https://deploy-preview-2664--oak-web-application.netlify.thenational.academy/webinars?page=1
https://deploy-preview-2664--oak-web-application.netlify.thenational.academy/teachers/programmes/english-secondary-ks3-l/units
https://deploy-preview-2664--oak-web-application.netlify.thenational.academy/teachers/search?term=science

3. You should see new `OakPagination` component has been implemented with smooth scrolling functionality

4. `PaginationHead` component adds next and prev into the head of page when previous or next pages are available 

## SEO Docs

https://www.notion.so/oaknationalacademy/SEO-recommendations-for-pagination-d69ae43e60be4485b3e061d76d0f4542

## Screenshots

How it used to look (delete if n/a):

https://github.com/user-attachments/assets/e8103a95-d1d3-4dfd-92fc-069e71062de8


<img width="996" alt="Screenshot 2024-08-05 at 11 52 42" src="https://github.com/user-attachments/assets/aa0b7a64-3ac2-488a-9ca0-c4126795d4f1">
<img width="671" alt="Screenshot 2024-08-05 at 11 53 20" src="https://github.com/user-attachments/assets/9ea5517a-3d4c-452e-99c4-fec10acc6fe6">


How it should now look:

<img width="693" alt="Screenshot 2024-08-05 at 11 43 23" src="https://github.com/user-attachments/assets/beddd2c1-9375-4e33-adeb-8db900002839">

https://github.com/user-attachments/assets/d48616af-7cfa-4871-a741-dcd52172f07f


## Checklist
